README.md

```diff
@@ -1,2 +1,185 @@
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Hello_microverse <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences.
+
+**[Hello Microverse]** is an initiation project to show micronauts on how to build future projects. It uses linters to check for errors and it contains an html file that displays an header of "Hello Microverse!". It also contains a css file thaat styles the header with a color of blue. It also contains a gitignore file that contains folders and files that are to be ignoreed by git.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">HTML & CSS</a></li>
+  </ul>
+</details>
+
+<!-- <details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details> -->
+
+<!-- <details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details> -->
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **Displays an text of Hello Microverse**
+- **Styled the text with a color of blue**
+- **Ignored node_modules and test.md**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+- **Click on the the code button on this repository to copy the links**
+- **copy either the SSH link or the HTTPS link**
+- **Open the terminal in you text editor**
+- **Navigate using the command line command to a prefered directory to save the project folder**
+- **Type in "git clone <clone-link>"**
+- **Then open the index.html file**
+- **Right click onn the interface and click on "open in browser"**
+
+
+
+### Prerequisites
+
+In order to run this project you need:
+
+VScode/Bracket/Sublime/Atom
+A Browser
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
```

Comment:
- [ ] In general your readme file is missing a few headers which are requirements for a professional readme, rather than removing them completing i would like to please ask you to add them back and customise them in a professional manner so that even non developers can pick up your code and run your project without having used github before. :+1: 
- [ ] you ommitted the following sections `install, usuage, deployments, run tests` so please see this [readme template](https://github.com/microverseinc/readme-template) and add them back and customise them using some of the information you already provided in the `get started` section :+1:

Reply:
I was asked by my previous reviewer to remove sections that are not relevant to my project. Since there's no installation, deployments necessary in this project, can i ommit it?

Reply:
Hi @Ridalord ,you need to add the installation section.Please add all the tools you installed during the project implementation such as Linters etc. The deployment section is not compulsory for this milestone of your project since there's no deployment done.